logs/logs.go

```diff
@@ -0,0 +1,56 @@
+package logs
+
+import (
+	jwt "github.com/dgrijalva/jwt-go"
+	dockerClient "github.com/fsouza/go-dockerclient"
+	"github.com/rancherio/host-api/app/common/connect"
+	"github.com/rancherio/host-api/config"
+	"net/http"
+	"strconv"
+)
+
+func GetLogs(rw http.ResponseWriter, req *http.Request) error {
+	urlToken := req.URL.Query().Get("token")
+	token, err := jwt.Parse(urlToken, func(token *jwt.Token) (interface{}, error) {
+		return token, nil
+	})
+
+	exec := token.Claims["exec"].(map[string]interface{})
+	container := exec["Container"].(string)
+	follow, found := exec["Follow"].(bool)
+	if !found {
+		follow = true
+	}
+	tailTemp, found := exec["Lines"].(float64)
```

Comment:
That is just the only way it works when pulling it in with jwt. Couldnt get others to work.